internal/handlers/common/filter.go

```diff
@@ -512,7 +512,10 @@ func filterFieldExpr(doc *types.Document, filterKey string, expr *types.Document
 func filterFieldRegex(fieldValue any, regex types.Regex) (bool, error) {
 	re, err := regex.Compile()
 	if err != nil {
-		return false, err
+		return false, NewError(
+			ErrRegexMissingParen,
+			err,
+		)
```

Comment:
#593

Reply:
There was long `fmt.Sprintf` line before. Forgot to format that piece.